docs/features/ge_for_teams.rst

```diff
@@ -0,0 +1,60 @@
+.. _ge_for_teams:
+
+##################
+GE for Teams
+##################
+
+Now that you've enjoyed single player mode, let's bring Great Expectations to
+your team.
+
+When you move from single user to collaborative use there are a few major things
+to consider.
+
+Where Should Expectations Live?
+===================================
+
+If you followed our best practice recommendation of committing the
+``great_expectations`` directory to your source control repository, then this
+question is already answered! Expectations live right in your repo!
+
+Where Should Validations Live?
+===================================
+
+When using the default Validation Operators, Validations are stored in your
+``great_expectations/uncommitted/validations/`` directory. Because these may
+examples of data (which could be sensitive or regulated) these Validations
```

Comment:
"Because these may examples of data" -> "Because these may contain examples of data"

Reply:
ack good catch.